examples/gridfs_upload.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This example demonstrates how to upload and download files from a stream with GridFS.
+ */
+
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use MongoDB\BSON\ObjectId;
+use MongoDB\Client;
+
+use function assert;
+use function fclose;
+use function fopen;
+use function fwrite;
+use function getenv;
+use function rewind;
+use function stream_get_contents;
+
+use const PHP_EOL;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$client = new Client(getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/');
+
+$gridfs = $client->selectDatabase('test')->selectGridFSBucket();
+
+// Create an in-memory stream, this can be any stream source like STDIN or php://input for web requests
+$stream = fopen('php://temp', 'w+');
+fwrite($stream, 'Hello world!');
+rewind($stream);
+
+// Upload to GridFS from the stream
+$id = $gridfs->uploadFromStream('hello.txt', $stream);
+assert($id instanceof ObjectId);
+echo 'Inserted file with ID: ' . $id . PHP_EOL;
```

Comment:
`gridfs_stream_wrapper.php` appears to be the first file that used `PHP_EOL`. Every other examples uses newline characters. Since the expected output in ExamplesTest uses Unix-style line endings, I think `"\n"` is the optimal thing to use in output.

Reply:
Ok, fixed for the 3 gridfs examples.